.devcontainer/conda-forge/devcontainer.json

```
@@ -0,0 +1,41 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+{
+    "name": "Miniforge (default-channel=conda-forge)",
+    "image": "condaforge/miniforge3:latest",
+
+    // Features to add to the dev container. More info: https://containers.dev/features.
+    // "features": {},
+
+    // Use 'forwardPorts' to make a list of ports inside the container available locally.
+    // "forwardPorts": [],
+    "mounts": [
+        "source=${localWorkspaceFolder}/../conda,target=/workspaces/conda,type=bind,consistency=cached",
+        "source=${localWorkspaceFolder}/../mamba,target=/workspaces/mamba,type=bind,consistency=cached"
+    ],
+    // Use 'postCreateCommand' to run commands after the container is created.
+    "postCreateCommand": "bash /workspaces/conda-libmamba-solver/.devcontainer/post_create.sh",
+    // Use 'postStartCommand' to run commands after the container is started.
+    "postStartCommand": "bash /workspaces/conda-libmamba-solver/.devcontainer/post_start.sh",
+
+    // Configure tool-specific properties.
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "python.testing.pytestArgs": [
+                    "tests"
+                ],
+                "python.testing.unittestEnabled": false,
+                "python.testing.pytestEnabled": true
+            },
+            "extensions": [
+                "charliermarsh.ruff",
+                "eamodio.gitlens",
+                "ms-toolsai.jupyter"
+            ]
+        }
+    }
+
+// Adjust to connect as non-root instead. More info: https://aka.ms/dev-containers-non-root.
```

Indentation mishap; probably my fault, sorry.
```suggestion
    // Adjust to connect as non-root instead. More info: https://aka.ms/dev-containers-non-root.
```